### ISSUE TITLE
Request textDocument/hover failed with message: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by daimor** [intersystems/language-server#20]
Code 
```objectscript
do ##class(%OAuth2.JWT).ObjectToJWT(.JOSE, obj, jwks, jwks, .JWT)
```
Hover on method name fails with errors
```
[Error - 3:25:03 PM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'length' of undefined
  Code: -32603 
```

**Comment by isc-bsaviano** [comment]
I can't reproduce either of these errors. For the code in the first comment, hover returns the correct documentation for the class and method. For the code in the second comment, hover correctly returns nothing. Is there any more context that might help?

**Comment by daimor** [comment]
So, I've managed to narrow the issue, and it happens due to security limitations, for users with no %All access.
And I see that you have a bunch of queries to %Dictionary classes. While default security settings for role %Developer does not allow such access, we should provide information for users, how to configure it.

And for sure, the extension should catch such errors.

```
statusCode: 200,
....
data: {
    status: {
      errors: [Array],
      summary: 'ERROR #5540: SQLCODE: -99 Message: User demo is not privileged for the operation'
    },
    console: [],
    result: {}
  }
```

**Comment by isc-bsaviano** [comment]
Thanks for investigating. My code does catch errors but this slips by because the HTTP response is 200OK despite the query failing. I've added in some more checks so you shouldn't get the error message from the LS.

**Comment by isc-bsaviano** [comment]
Fixed in version 1.0.0-beta.5. Also added documentation for how to grant the correct privileges in the README.
